src/DIPS.Xamarin.UI/Controls/Toast/Toast.cs

```diff
@@ -0,0 +1,274 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using DIPS.Xamarin.UI.Internal.Xaml;
+using Xamarin.Essentials;
+using Xamarin.Forms;
+
+namespace DIPS.Xamarin.UI.Controls.Toast
+{
+    /// <summary>
+    ///     Toast control that would appear on top of the presented view
+    /// </summary>
+    public class Toast : BindableObject
```

Comment:
Does this need to inherit from `BindableObject`? The reason I ask is because this is the API options the consumer get's when using the API:
![image](https://user-images.githubusercontent.com/2527084/89189664-9a64c380-d5a0-11ea-8a75-a234d9058378.png)
I was expecting this to be very clean with only `DisplayToast`

Reply:
Oh, I removed using all Bindings from the toast, however I've missed to remove the extend